digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -212,34 +212,39 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
 
   def municipalityDao = new MunicipalityDao
 
-  private def getMapViewStartParameters(mapView: Option[MapViewZoom]): Option[(Double, Double, Int)] = mapView.map(m => (m.geometry.x, m.geometry.y, m.zoom))
+  private def getMapViewStartParameters(mapView: Option[MapViewZoom], assetType: Int): Option[(Double, Double, Int, Int)] = mapView.map(m => (m.geometry.x, m.geometry.y, m.zoom, assetType))
 
 
   get("/startupParameters") {
-    val defaultValues: (Double, Double, Int) = (390000, 6900000, 2)
+    val defaultValues: (Double, Double, Int, Int) = (390000, 6900000, 2, 10)
     val user = userProvider.getCurrentUser()
-    val userPreferences: Option[(Long, Long, Int)] = (user.configuration.east, user.configuration.north, user.configuration.zoom) match {
-      case (Some(east), Some(north), Some(zoom)) => Some(east, north, zoom)
+    val assetTypeId = user.configuration.assetType match {
+      case Some(assetType) => assetType
+      case _  => 10
+    }
+    val userPreferences: Option[(Long, Long, Int, Int)] = (user.configuration.east, user.configuration.north, user.configuration.zoom, user.configuration.assetType) match {
+      case (Some(east), Some(north), Some(zoom), None) => Some(east, north, zoom, 10)
+      case (Some(east), Some(north), Some(zoom), Some(assetType)) => Some(east, north, zoom, assetType)
```

Comment:
Since you are getting the assetTypeId in the previous line, this code could be replace by
```
val userPreferences: Option[(Long, Long, Int)] = (user.configuration.east, user.configuration.north, user.configuration.zoom) match {	  
  val assetTypeId = user.configuration.assetType match {
      case (Some(east), Some(north), Some(zoom)) => Some(east, north, zoom, assetTypeId )
```